ocrd_tesserocr/recognize.py

```diff
@@ -74,6 +77,12 @@ class TesserocrRecognize(Processor):
     def __init__(self, *args, **kwargs):
         kwargs['ocrd_tool'] = OCRD_TOOL['tools'][TOOL]
         kwargs['version'] = OCRD_TOOL['version'] + ' (' + tesseract_version().split('\n')[0] + ')'
+        if kwargs.get('list_resources', False):
```

Comment:
What about `show_resource`? We cannot use the superclass's constructor for that either, because `list_resource_candidates` does not apply...

Reply:
Ah, right, I did test it but not with the `TESSDATA_PREFIX` said. I'll add that.

Reply:
added in 59bb925. If you have an idea how the commented-out test might be coaxed into actually capturing STDOUT, pls let me know.

Reply:
> If you have an idea how the commented-out test might be coaxed into actually capturing STDOUT, pls let me know.

I guess `TestCase.invoke_cli` expects line-buffered output, whereas `shutil.copyfileobj` gives us unbuffered output.

Reply:
But TBH I don't understand how these Pytest capturing fixtures (your use of `capfd` in `invoke_cli`) work.

Reply:
I tried various possible fixes but to no avail. It doesn't make sense because we use the same pattern e.g. in `tests/cli/test_resmgr.py` in OCR-D/core and there are no issues there, they're using the same click and pytest.

But I verified manually that the specified resource is properly written to STDOUT for `-C` and the list is correct for `-L`.

Perhaps @M3ssman, being the pytest expert, has an idea?

But it should not stop us from releasing.